docs/en/connector-v2/sink/OssFile.md

```diff
@@ -0,0 +1,203 @@
+# OssFile
+
+> Oss file sink connector
+
+## Description
```

Comment:
Please update this document when https://github.com/apache/incubator-seatunnel/pull/2625 merged